onnxruntime/test/providers/cpu/tensor/pad_test.cc

```diff
@@ -809,22 +809,74 @@ TEST(PadOpTest, ConstantPadAxes) {
   OpTester test("Pad", 18);
   test.AddAttribute("mode", "constant");
   test.AddInput<int32_t>("data", {1, 2, 2, 2},
-  {
-    1, 1,
-    1, 1,
-    1, 1,
-    1, 1});
+                         {1, 1,
+                          1, 1,
+                          1, 1,
+                          1, 1});
   test.AddInput<int64_t>("pads", {4}, {0, 1, 0, 1});
   test.AddInput<int32_t>("value", {1}, {0});
+#if defined(USE_COREML) && defined(__APPLE__)
```

Comment:
nit: could refactor so we always test with axes as an initializer, and skip the second test if it's coreml.